src/main/scala/is/hail/methods/SplitMulti.scala

```diff
@@ -44,16 +44,17 @@ object SplitMulti {
     val splitGenotypeStreamBuilders = splitVariants.map { case (sv, _) => new GenotypeStreamBuilder(sv.nAlleles, isLinearScale) }
 
     for (g <- it) {
-
-      val gadsum = g.ad.map(gadx => (gadx, gadx.sum))
+      val gadsum = Genotype.ad(g).map(gadx => (gadx, gadx.sum))
```

Comment:
Me too, me too.  I basically think we should rip out Genotype and then worry about how to speed up generic genotypes.